canonical/ubuntu.com
# WD-10166 Update certified image

## Done

- Update image as described in  https://warthogs.atlassian.net/browse/WD-10166
- Djlint formatting

## QA

- Go to https://ubuntu-com-13753.demos.haus/certified and check that the image has been updated according [Jira ticket](https://warthogs.atlassian.net/browse/WD-10166)


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10166

## Screenshots

<img width="1270" alt="Screenshot 2024-04-16 at 11 19 15" src="https://github.com/canonical/ubuntu.com/assets/14939793/88ef9fd3-cf90-4780-9c7a-ac47c26cc4f5">



## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
